client/src/component/Overview/Product-Info/ProductDescription.jsx

```diff
@@ -2,22 +2,31 @@ import React from 'react';
 import styled from 'styled-components';
 
 const ProductDescription = ({ currentProduct }) => (
-  <div>
-    <SloganStyled>{currentProduct.slogan}</SloganStyled>
-    <DescriptionStyled>{currentProduct.description}</DescriptionStyled>
-  </div>
+  <Paragraph>
+    <div>
```

Comment:
if Paragraph is already a styled div having an anonymous div inside of it seems superfluous to me. Gas no bearing on functionality, however, so I'll leave it up to you as to whether it's good to PR or not.